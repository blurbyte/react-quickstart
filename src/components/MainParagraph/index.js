import styled from 'styled-components';

const MainParagraph = styled.p`
  font-family: 'OpenSans Light', Arial, Helvetica, sans-serif;
  font-size: 2.4rem;
  color: #444;
  line-height: 1.4;
  margin: 0.8rem 0;
`;

export default MainParagraph;
