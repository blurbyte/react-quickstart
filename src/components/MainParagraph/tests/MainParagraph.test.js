import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import MainParagraph from '../index';

describe('<MainParagraph />', () => {
  it('should render correctly', () => {
    // component init
    const component = renderer.create(
      <MainParagraph />
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
