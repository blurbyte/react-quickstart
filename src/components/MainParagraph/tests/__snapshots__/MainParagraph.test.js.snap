// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainParagraph /> should render correctly 1`] = `
.lmNQTu {
  font-family: 'OpenSans Light', Arial, Helvetica, sans-serif;
  font-size: 2.4rem;
  color: #444;
  line-height: 1.4;
  margin: 0.8rem 0;
}

<p
  className="sc-bdVaJa lmNQTu"
/>
`;
